src/views/ScaleSlider.js

```diff
@@ -0,0 +1,80 @@
+const BoneView = require("backbone-viewj");
+const _ = require("underscore");
+const $ = require("jbone");
+//const Slider = require("bootstrap-slider");
+
+const View = BoneView.extend({
+
+  initialize: function(data) {
+    this.g = data.g;
+    this.listenTo(this.g.zoomer,"change:columnWidth", this.render);
+    return this;
+  },
+  
+  attributes: {
+    class: "biojs_msa_scale"
+  },
+  
+  events: {
+    'change input': 'updateSlider',
+    'click button': 'clickButton',
+  },
+  
+  template: _.template('\
+<input type="range" \
+  data-provide="slider" \
+  min="<%= min %>" \
+  max="<%= max %>" \
+  step="<%= step %>" \
+  value="<%= value %>" \
+>\
+<div class="btngroup msa-btngroup">\
+<button class="btn msa-btn" data-action="smaller"><span class="glyphicon-zoom-out"></span>-</button>\
```

Comment:
this was working on the basis that it will play nicely with bootstrap3 css (if they are included)